packages/initrd/cos-config-launcher.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
This hook is damn ugly, probably would be nicer to just install cos-setup inside the initrd and create an early service that loads COS_OEM volume if any and run initramfs.after from the initramfs itself. But this might be a more sophisticated approach to do `ignition` or `combustion` like tweaks.

Reply:
I'd say it's fine anyway, it opens up for further tweaks that we might need really soon